cognition_objects/environment_variable.py

```diff
@@ -46,13 +48,27 @@ def get_by_name(
     )
 
 
+def get_by_name_and_org_id(
+    org_id: str,
+    name: str,
+) -> CognitionEnvironmentVariable:
+
+    return (
+        session.query(CognitionEnvironmentVariable)
+        .filter(CognitionEnvironmentVariable.organization_id == org_id)
+        .filter(CognitionEnvironmentVariable.project_id == None)
+        .filter(CognitionEnvironmentVariable.name == name)
```

Comment:
usually we use filter with multiple comma sperated filters for AND connected ones we dont need to extend afterwards